websites/0-9/4chan/presence.ts

```diff
@@ -0,0 +1,123 @@
+const presence = new Presence({
+	clientId: "1195051461822914640",
+});
+
+const enum Assets {
+	Cog = "https://i.imgur.com/XAgEs5E.png",
```

Comment:
It must be named Logo & also must be 512x512 just like the metadata logo
```suggestion
	Logo = "https://i.imgur.com/7pI9X5M.png",
```